ingest-app/src/cmr/ingest/services/ingest_service/variable.clj

```diff
@@ -1,15 +1,39 @@
 (ns cmr.ingest.services.ingest-service.variable
   (:require
+   [cheshire.core :as json]
+   [clojure.string :as string]
    [cmr.common.util :refer [defn-timed]]
    [cmr.transmit.metadata-db2 :as mdb2]
-   [cmr.umm-spec.umm-spec-core :as spec]))
+   [cmr.umm-spec.umm-spec-core :as spec]
+   [digest]))
```

Comment:
We should probably have `[digest :as digest]` even though that seems redundant just to be consistent. But anyway I think this should just go away and define the md5 function ourselves.

Reply:
Done.